backend/src/main/kotlin/blue/mild/covid/vaxx/service/PatientService.kt

```diff
@@ -47,13 +51,32 @@ class PatientService {
     suspend fun getAllPatients(): List<PatientDtoOut> =
         newSuspendedTransaction { getAndMapPatients() }
 
-    suspend fun getPatientsByPersonalNumber(patientPersonalNumber: String): List<PatientDtoOut> =
-        newSuspendedTransaction { getAndMapPatients { Patient.personalNumber eq patientPersonalNumber } }
+    suspend fun getPatientsByPersonalNumber(patientPersonalNumber: String): List<PatientDtoOut> {
+        validationService.validatePersonalNumberAndThrow(patientPersonalNumber)
+        return newSuspendedTransaction { getAndMapPatients { Patient.personalNumber eq patientPersonalNumber } }
+    }
 
-    suspend fun getPatientsByEmail(email: String): List<PatientDtoOut> =
-        newSuspendedTransaction { getAndMapPatients { Patient.email eq email } }
+
+    suspend fun getPatientsByEmail(email: String): List<PatientDtoOut> {
+        validationService.validateEmailAndThrow(email)
```

Comment:
je to tady nutny? tohle je get, u nej nam je to prece jedno, jestli to je validni nebo ne, bud to neco najde v db nebo ne

Reply:
Hele, proc ne, kdyz uz to mame....

Reply:
ok no, zatim nechme, mozna to nekdy vyhodim